bin/test-lxd-storage-vm

```diff
@@ -420,6 +420,57 @@ do
         lxc image delete vmbig
         lxc profile delete vmsmall
 
+        echo "==> Checking VM UUID while snapshotting (non-stateful mode)"
+        lxc init images:ubuntu/20.04/cloud v1 --vm -s "${poolName}"
+        lxc start v1
+        waitVMAgent v1
+        lxc info v1
+
+        initialUUID=$(lxc config show v1 --expanded | grep "volatile.uuid:" | awk '{print $2}')
```

Comment:
`lxc config get v1 volatile.uuid` ...again ;)

Reply:
ah yeah I missed this one